.travis.yml

```diff
@@ -10,7 +10,7 @@ stages:
   - test
 
 before_script:
-  - bundle exec danger
+  - '[ ! -z $DANGER_GITHUB_API_TOKEN ] && bundle exec danger || echo "Skipping Danger for External Contributor"'
```

Comment:
Would something like this be reasonable to put in our Dangerfile instead? That way if you try to run it locally you'd get some indication that you need to set `DANGER_GITHUB_API_TOKEN` to proceed.

Reply:
I had considered this as well, but afaik danger will error before it even gets to the Dangerfile if this variable is not set.